packages/flutter/lib/src/widgets/locale_query.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'framework.dart';
 
-// Superclass for locale-specific data provided by the application.
+/// Superclass for locale-specific data provided by the application.
 class LocaleQueryData { }
 
 class LocaleQuery<T extends LocaleQueryData> extends InheritedWidget {
```

Comment:
I meant the LocaleQuery class, not just LocaleQueryData. But yes, there's not really any reason for T here to be required to extend LocaleQueryData.